pkgs/development/libraries/pipewire/default.nix

```diff
@@ -181,7 +167,6 @@ let
       "-Dgstreamer=${mesonEnableFeature gstreamerSupport}"
       "-Dsystemd-system-service=${mesonEnableFeature enableSystemd}"
       "-Dudev=${mesonEnableFeature (!enableSystemd)}"
-      "-Dudevrulesdir=${placeholder "out"}/lib/udev/rules.d"
```

Comment:
Ditto here, if you have a link to upstream meson.build changes that make this line unnecessary I'd appreciate seeing it.

Reply:
That is still necessary, I think it got lost in the rebases somewhere. I'll put it back when I'm back home in like an hour.

Reply:
Wait no never mind this is fine. It was just a duplicated line: https://github.com/NixOS/nixpkgs/pull/262148/files#diff-fdd948de77300ac589c79c5b6dd5f79e757362ad88beac84d4352da1aa7bfa12R158

Reply:
That duplicate is missing the "out" placeholder though. Not sure if that is needed any more.

Reply:
It isn't, Meson will treat relative paths as relative to the prefix (which is `$out`).